src/commands/pg/locks.ts

```diff
@@ -0,0 +1,50 @@
+'use strict'
+
+import {utils} from '@heroku/heroku-cli-util'
+import {Command, flags} from '@heroku-cli/command'
+import {Args, ux} from '@oclif/core'
+
+export const generateLocksQuery = (truncate: boolean): string => {
+  const querySnippet = truncate
+    ? 'CASE WHEN length(pg_stat_activity.query) <= 40 THEN pg_stat_activity.query ELSE substr(pg_stat_activity.query, 0, 39) || \'...\' END'
+    : 'pg_stat_activity.query'
+
+  return `
+  SELECT
+    pg_stat_activity.pid,
+    pg_class.relname,
+    pg_locks.transactionid,
+    pg_locks.granted,
+    ${querySnippet} AS query_snippet,
+    age(now(),pg_stat_activity.query_start) AS 'age'
```

Comment:
```suggestion
    age(now(),pg_stat_activity.query_start) AS "age"
```
For SQL this does need to be double quotes.